### PR TITLE
Docker correction -jekyll

### PR DESCRIPTION
**Pull request by finp**
some updates to docker option

### CODE REVIEW

**Comment by jstaffor** [comment]
Dockerfile - Line 9 | Something I encountered:

[jstaffor@localhost docker]$ useradd -d /home/aerogear -G rvm aerogear
useradd: group 'rvm' does not exist

File: docker/README.md [hunk 1]
```diff
@@ -5,11 +5,19 @@ You may build this via
...
 ```
 
+To run the container:
+
+```
+./run.sh
+```
+from this directory (does not work if you run it from a different directory).
+
+
```
Comment (secondsun) [inline docker/README.md]: Here we should insert a new section "Development" or something similar.  I don't feel that the flow from "here is a script" to "here is a long docker command" flows particularly well.

Optionally run.sh could be edited to take a --mount option which will mount the aerogear directory it is currently in.

**Comment by secondsun** [comment]
Error When I try to build 

```
Step 19/21 : RUN /bin/bash -l -c "bundle install --path vendor"
 ---> Running in 6b1885e5f4e0
Fetching gem metadata from https://rubygems.org/..............
Fetching version metadata from https://rubygems.org/...
Fetching dependency metadata from https://rubygems.org/..
Resolving dependencies...
Fetching RedCloth 4.2.9
Installing RedCloth 4.2.9 with native extensions
Fetching concurrent-ruby 1.0.5
Installing concurrent-ruby 1.0.5
Fetching i18n 0.8.6
Installing i18n 0.8.6
Fetching minitest 5.10.3
Installing minitest 5.10.3
Fetching thread_safe 0.3.6
Installing thread_safe 0.3.6
Fetching public_suffix 2.0.5
Installing public_suffix 2.0.5
Fetching asciidoctor 1.5.3
Installing asciidoctor 1.5.3
Using bundler 1.15.3
Fetching colorator 0.1
Installing colorator 0.1
Fetching colored 1.2
Installing colored 1.2
Fetching commonjs 0.2.7
Installing commonjs 0.2.7
Fetching curb 0.9.3
Installing curb 0.9.3 with native extensions
Fetching diskcached 1.1.3
Installing diskcached 1.1.3
Fetching ffi 1.9.18
Installing ffi 1.9.18 with native extensions
Fetching mercenary 0.3.6
Installing mercenary 0.3.6
Fetching mini_portile2 2.2.0
Installing mini_portile2 2.2.0
Fetching parallel 1.12.0
Installing parallel 1.12.0
Fetching yell 2.0.7
Installing yell 2.0.7
Fetching rb-fsevent 0.10.2
Installing rb-fsevent 0.10.2
Fetching kramdown 1.6.0
Installing kramdown 1.6.0
Fetching liquid 3.0.6
Installing liquid 3.0.6
Fetching rouge 1.8.0
Installing rouge 1.8.0
Fetching safe_yaml 1.0.4
Installing safe_yaml 1.0.4
Fetching libv8 3.16.14.19 (x86_64-linux)
Installing libv8 3.16.14.19 (x86_64-linux)
Fetching net-ssh 4.1.0
Installing net-ssh 4.1.0
Fetching ref 2.0.0
Installing ref 2.0.0
Fetching tzinfo 1.2.3
Installing tzinfo 1.2.3
Fetching addressable 2.5.1
Installing addressable 2.5.1
Fetching less 2.6.0
Installing less 2.6.0
Fetching ethon 0.10.1
Installing ethon 0.10.1
Fetching rb-inotify 0.9.10
Installing rb-inotify 0.9.10
Fetching nokogiri 1.8.0
Installing nokogiri 1.8.0 with native extensions
Fetching net-sftp 2.1.2
Installing net-sftp 2.1.2
Fetching therubyracer 0.12.1
Installing therubyracer 0.12.1 with native extensions
Fetching activesupport 5.1.3
Installing activesupport 5.1.3
Fetching typhoeus 0.8.0
Installing typhoeus 0.8.0
Fetching sass-listen 4.0.0
Installing sass-listen 4.0.0
Fetching listen 3.0.8
Installing listen 3.0.8
Gem::InstallError: activesupport requires Ruby version >= 2.2.2.
An error occurred while installing activesupport (5.1.3), and Bundler cannot
continue.
Make sure that `gem install activesupport -v '5.1.3'` succeeds before bundling.

In Gemfile:
  html-proofer was resolved to 3.7.2, which depends on
    activesupport
The command '/bin/sh -c /bin/bash -l -c "bundle install --path vendor"' returned a non-zero code: 5
```

File: docker/README.md (hunk 1 above)
Comment (finp) [inline docker/README.md]: @secondsun wanna keep run.sh simple, but you're right, it doesn't flow, will add 'Advanced' section.

**Comment by finp** [comment]
@jstaffor @pmacko1  I cannot easily reproduce Summer's issue, could you try, ie first instruction in /docker/README.md for this repo

**Comment by jstaffor** [comment]
(1) Extract from error message
Installing listen 3.0.8
Gem::InstallError: activesupport requires Ruby version >= **2.2.2**.

(2) Line 18 of file (https://github.com/aerogear/aerogear.org/blob/master/docker/Dockerfile)
"  RUN  /bin/bash -l -c "rvm install ruby **2.1.2**"

**Comment by finp** [comment]
cannot replicate my issue, and cannot justify the effort to use this after getting jekyll working with rvm

**Pull request closed without merging**